quic/s2n-quic-core/src/recovery/pacing.rs

```diff
@@ -57,6 +61,11 @@ impl Pacer {
         max_datagram_size: u16,
         slow_start: bool,
     ) {
+        if rtt_estimator.smoothed_rtt() < MINIMUM_PACING_RTT {
+            self.next_packet_departure_time = None;
```

Comment:
If we increase the `INITIAL_INTERVAL` > 0, this would mean the first packet sent after `smoothed_rtt` exceeds `MINIMUM_PACING_RTT` would have the `INITIAL_INTERVAL` applied, when that is intended only for the second packet on a path. So you could do `self.next_packet_departure_time = now`, or just not set it as you were doing. Otherwise we would probably want some additional state to know when we are at the very beginning of a path's existence.

Reply:
I can just remove the set like before. I was going off of your comment:

>  I was a little uneasy with how this would just not be updating self.next_packet_departure_time